src/libsync/utility.cpp

```diff
@@ -424,6 +424,21 @@ bool Utility::isBSD()
 #endif
 }
 
+QString Utility::platformName()
```

Comment:
Thanks @guruz of course super Qt comes with something awesome. See additional patch.